### DISCUSSION TITLE
[Feature] Bulk change Timezone

### DISCUSSION THREAD

**Discussion by TrueDoctor** [immich-app/immich#14615] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When traveling abroad, I sometimes forget to change the timezone on my camera. When displaying the camera photos in a timeline together with phone pictures, everything becomes offset. It would be great if I could select all affected images (through filtering by time and camera model) and change the timezone for all of those pictures.
How is this different from the existing Edit Date functionality?
The existing feature sets the same date / time for all the images instead of offsetting them. 

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#9430

**Reply by TrueDoctor** [reply]
Ah, thanks, sorry for the duplicate, I searched for timezone and did not find anything relevant

**Comment by amtready** [comment]
I have exactly this request of bulk change of timezone. I don't want to move time X hours up or down. I just want to keep the hour as it it registered in the image file but just changing the timezone. As an example: I have a bunch of pictures captured the same day, but some of them didn't have the capture date exif field filled. So, I filled it using an script, but this one uses always UTC+0:00 to write the date. So, when uploading to Immich, I have pictures in UTC+2:00 and others in UTC+0:00, which makes to have them a bit disordered. So, it would be great, the same way we have "Change date" when having a selection of pics, to have a "Change timezone" so I could change only the timezone without altering the hh:mm:ss registed in those pictures.

I'm doing it now one by one in immich, which is quite... slow xD
